shell/platform/embedder/embedder.h

```diff
@@ -2582,9 +2582,23 @@ FlutterEngineResult FlutterEngineRunInitialized(
 //------------------------------------------------------------------------------
 /// @brief      Adds a view.
 ///
-///             This is an asynchronous operation. The view should not be used
-///             until the |add_view_callback| is invoked with an `added` of
-///             `true`.
+///             This operation notifies the engine to allocate resources and
+///             inform Dart about the view, and on success, schedules a new
```

Comment:
I would remove this bit:

> This operation notifies the engine to allocate resources and inform Dart about the view

In my opinion those are details unimportant to an embedder and are already implied by "Adds a view" above.